### PR TITLE
Fix room version conflict

### PR DESCRIPTION
**Pull request by alfonsorojasrvz**

### CODE REVIEW

File: app/src/main/res/layout/activity_main.xml [hunk 1]
```diff
@@ -4,90 +4,76 @@
...
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toBottomOf="@id/button" />
+
 
-    <androidx.recyclerview.widget.RecyclerView
+    <TextView
+        android:id="@+id/tvNoRecordsAvailable"
         android:layout_width="match_parent"
         android:layout_height="wrap_content"
-        tools:visibility="visible"
```
Comment (alfonsorojasrvz) [inline app/src/main/res/layout/activity_main.xml:71]: I removed this recyclerview cause it was crashing. You can add it back properly

File: app/src/main/res/layout/activity_main.xml [hunk 2]
```diff
@@ -4,90 +4,76 @@
...
-        app:layout_constraintStart_toStartOf="parent"
-        app:layout_constraintEnd_toEndOf="parent"
         android:layout_marginTop="50dp"
-        />
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toBottomOf="@id/button" />
+
 
-    <androidx.recyclerview.widget.RecyclerView
```
Comment (alfonsorojasrvz) [inline app/src/main/res/layout/activity_main.xml:68]: I removed this recyclerview. You can add it back properly

File: app/build.gradle [hunk 3]
```diff
@@ -5,12 +5,12 @@ plugins {
...
 android {
-    compileSdk 33
+    compileSdk 34
 
     defaultConfig {
-        applicationId "dev.loveth.roomdemo"
+        applicationId "com.example.roomdemo"
         minSdk 21
-        targetSdk 33
+        targetSdk 34
```
Comment (alfonsorojasrvz) [inline app/build.gradle:13]: target 34 was needed to make the alpha version work

File: app/build.gradle [hunk 4]
```diff
@@ -5,12 +5,12 @@ plugins {
 }
 
 android {
-    compileSdk 33
+    compileSdk 34
 
     defaultConfig {
-        applicationId "dev.loveth.roomdemo"
+        applicationId "com.example.roomdemo"
```
Comment (alfonsorojasrvz) [inline app/build.gradle:11]: wtf was this shit?

File: app/build.gradle [hunk 5]
```diff
@@ -53,12 +53,11 @@ dependencies {
     //kotlin extension for coroutine support with activities
     implementation "androidx.activity:activity-ktx:$activityVersion"
 
-
     implementation 'androidx.core:core-ktx:1.10.1'
     implementation 'androidx.appcompat:appcompat:1.6.1'
     implementation 'com.google.android.material:material:1.9.0'
     implementation 'androidx.constraintlayout:constraintlayout:2.1.4'
-    testImplementation 'junit:junit:'
+    testImplementation 'junit:junit:4.13.2'
```
Comment (alfonsorojasrvz) [inline app/build.gradle:60]: You were missing version number here

File: app/build.gradle [hunk 6]
```diff
@@ -35,14 +35,14 @@ android {
...
-        namespace 'com.example.namespace'
+        namespace 'com.example.roomdemo'
     }
 }
 
 
 dependencies {
-    def room_version = '2.3.0'
-    def activityVersion = '1.3.1'
+    def room_version = '2.6.0-alpha02'
```
Comment (alfonsorojasrvz) [inline app/build.gradle:44]: using the alfa version made it work. I also upgraded some other

File: app/build.gradle [hunk 7]
```diff
@@ -35,14 +35,14 @@ android {
...
+    def activityVersion = '1.7.2'
     def lifecycle_version = "2.4.0-beta01"
```
Comment (alfonsorojasrvz) [inline app/build.gradle:46]: U are not using this

File: app/build.gradle [hunk 8]
```diff
@@ -35,14 +35,14 @@ android {
     }
 
     android {
-        namespace 'com.example.namespace'
+        namespace 'com.example.roomdemo'
```
Comment (alfonsorojasrvz) [inline app/build.gradle:38]: also this was not correct, was messing with the imports in main activity

File: app/build.gradle (hunk 4 above)
Comment (Shomika994) [reply app/build.gradle:11]: hahah I have no idea, I copied whole buildgradle file at the end, cuz I was frustrated and thought at one point that there is the problem...

File: app/build.gradle (hunk 3 above)
Comment (Shomika994) [reply app/build.gradle:13]: I see

File: app/src/main/res/layout/activity_main.xml (hunk 2 above)
Comment (Shomika994) [reply app/src/main/res/layout/activity_main.xml:68]: Why ? What was wrong with recyclerview?

**Review by Shomika994 — APPROVED** [review]

File: app/src/main/res/layout/activity_main.xml (hunk 2 above)
Comment (alfonsorojasrvz) [reply app/src/main/res/layout/activity_main.xml:68]: The textview was in the middle of it

**Pull request merged**